src/app/features/posts/posts.module.ts

```diff
@@ -1,6 +1,6 @@
 import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
-import {PostsComponent} from './posts.component';
+import {PostsComponent} from './posts-container/posts.component';
```

Comment:
folder-name and component file name should be equal, recommend to rename posts to posts-container